web/public/js/services/user-utils.js

```diff
@@ -105,5 +105,22 @@ angular.module('cpZenPlatform').factory('userUtils', ['$location', '$window', '$
     return  avatar || overallDefault;
   }
 
+  userUtils.getTitleForUserTypes = function (userTypes) {
+    var title = [];
+    if (_.includes(userTypes, 'attendee-u13')) {
+      title.push($translate.instant('Ninja (Under 13)'));
```

Comment:
itd probably make more sense to actually just update them automatically in the backend rather then changing how the front end displays them